index.html

```diff
@@ -0,0 +1,85 @@
+</head>
+
+<body>
+
+    <article>
+        <a href="https://www.bbc.com/worklife/article/20200929-how-biophilic-design-can-create-a-better-workspace"
+            target="blank" <div class="image-wrapper">
+            <img src="https://cdn.glitch.com/5e9189cf-962e-4e13-a403-b7e677750267%2FcroppedTerracotta.jpg?v=1603939526784"
+                alt="green plants in terracotta pots on white shelves" />
+        </a>
+        </div>
+
+        <div class="article-details">
+            <ul class="u-list-reset u-list-inline">
+                <li>
+                    <a href="#">
+                        <svg xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 65.1 77.7"
+                            enable-background="new 0 0 65.1 77.7" xml:space="preserve">
+                            <path d="M8.7,72.7l21.5-24.9c1.2-1.4,3.5-1.4,4.8,0l21.5,24.9l3.5,4.1c1.6,1.9,5.2,0.9,5.2-1.4v-70c0-2.9-2.8-5.3-6.2-5.3H6.2
+                                C2.8,0,0,2.4,0,5.3v70c0,2.4,3.5,3.3,5.2,1.4L8.7,72.7z" alt="bookmark" />
+                        </svg>
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 85.27 81.68">
+                            <path stroke-miterlimit="10"
+                                d="M29.08 22.06L38.5 3.12a4.71 4.71 0 018.46 0l9.31 19a4.73 4.73 0 003.54 2.59l20.93 3.11a4.72 4.72 0 012.56 8.09L68.1 50.63a4.75 4.75 0 00-1.37 4.17l3.57 20.91a4.72 4.72 0 01-6.87 5l-18.7-9.9a4.73 4.73 0 00-4.39 0L21.58 80.6a4.72 4.72 0 01-6.83-5l3.64-20.84a4.71 4.71 0 00-1.34-4.18L1.94 35.76a4.72 4.72 0 012.64-8l21-3a4.72 4.72 0 003.5-2.7z"
+                                alt="star" />
+                        </svg>
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <svg xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 69.9 78"
+                            enable-background="new 0 0 69.9 78" xml:space="preserve">
+                            <path
+                                d="M55.2,48.5c-5.4,0-10,2.9-12.6,7.2l-14.9-9.6c1.1-2.1,1.8-4.5,1.8-7.1c0-1.7-0.3-3.4-0.9-4.9l15.5-9.6c2.7,3.1,6.6,5,11,5
+                                c8.1,0,14.7-6.6,14.7-14.7C69.9,6.6,63.3,0,55.2,0C47,0,40.4,6.6,40.4,14.7c0,1.9,0.4,3.8,1.1,5.5l-15.3,9.5
+                                c-2.7-3.3-6.8-5.4-11.4-5.4C6.6,24.3,0,30.9,0,39c0,8.1,6.6,14.7,14.7,14.7c3.8,0,7.2-1.4,9.8-3.8l16.2,10.5
+                                c-0.2,0.9-0.3,1.9-0.3,2.8C40.4,71.4,47,78,55.2,78c8.1,0,14.7-6.6,14.7-14.7C69.9,55.1,63.3,48.5,55.2,48.5z"
+                                alt="share" />
+                        </svg>
+                    </a>
+                </li>
+            </ul>
+            <h3>
+                <a class="cat-title" href="https://www.bbc.com/worklife/tags/working-from-home" target="blank">
```

Comment:
The `target` attribute here needs to start with an underscore, like so:
```suggestion
                <a class="cat-title" href="https://www.bbc.com/worklife/tags/working-from-home" target="_blank">
```